Blueprints/BlueprintDefinitions/vs2022/PowertoolsServerless/template/src/BlueprintBaseName.1/Functions.cs

```diff
@@ -0,0 +1,52 @@
+using System.Net;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.APIGatewayEvents;
+using AWS.Lambda.Powertools.Logging;
+using AWS.Lambda.Powertools.Metrics;
+using AWS.Lambda.Powertools.Tracing;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace BlueprintBaseName._1;
+
+/// <summary>
+/// Learn more about Lambda Powertools at https://awslabs.github.io/aws-lambda-powertools-dotnet/
+/// Lambda Powertools Logging: https://awslabs.github.io/aws-lambda-powertools-dotnet/core/logging/
+/// Lambda Powertools Tracing: https://awslabs.github.io/aws-lambda-powertools-dotnet/core/tracing/
+/// Lambda Powertools Metrics: https://awslabs.github.io/aws-lambda-powertools-dotnet/core/metrics/
+/// </summary>
+public class Functions
+{
+    /// <summary>
+    /// A Lambda function to respond to HTTP Get methods from API Gateway
+    /// </summary>
+    /// <param name="request"></param>
+    /// <returns>The API Gateway response.</returns>
+    [Logging(LogEvent = true, CorrelationIdPath = CorrelationIdPaths.ApiGatewayRest)]
+    [Metrics(CaptureColdStart = true)]
+    [Tracing(CaptureMode = TracingCaptureMode.ResponseAndError)]
+    public APIGatewayProxyResponse Get(APIGatewayProxyRequest request, ILambdaContext context)
+    {
+        Logger.LogInformation("Get Request\n");
```

Comment:
I know this is a copy and paste from another template were we should fix this but can you remove the `\n`? I'm not sure why we ever had that.